### ISSUE TITLE
Change App-Logo/Icon

### ISSUE THREAD

**Issue by stefan-niedermann** [owncloud/android#475]
The Android News-App gets a new icon (https://github.com/owncloud/News-Android-App/issues/199), the official android files app should to that, too.

this would look the android apps much more consistent, as all apps have the same scheme (blue gradien background with flat white logo). Owncloud Webinterface itself uses a white cloud on blue background, too.

I tried it with the grey cloud, too. But this didn't look very nice.

My suggestion is this:

![android-white](https://cloud.githubusercontent.com/assets/4741199/2798680/bd5db610-cc53-11e3-9fb4-c8769d31065a.png)

Opionions? @jancborchardt

**Comment by stefan-niedermann** [comment]
Hi, any progress here? Its really confusing to see different icon types for the official app and the eco system apps (e.g. news app).

> We already planned to use different icons in the desktop / launcher and in the app top bar with the next release of the app.

Didn't happen, did it?

**Comment by jancborchardt** [comment]
The one big reason I think the blue-background logo is much better than the white-background one is that it’s much more visible among other apps. The lightblue on white is just very difficult to make out. cc @MTRichards @karlitschek

**Comment by jancborchardt** [comment]
Every time I use my Android phone it’s so difficult to find the ownCloud app due to the hardly visible logo. Any call on this @MTRichards @karlitschek?

**Comment by davivel** [comment]
If there is agreement about the logo we can update it in the release 1.7.2 without risks.

@jancborchardt  @MTRichards @karlitschek

**Comment by MTRichards** [comment]
Doesn't it depend entirely on what your background is? If you have a light background, it is hard to see, if you have a dark background easy to see?
@jancborchardt can you pop a screenshot up here to illustrate the issue?

I have a dark background on Android and it looks just fine, but a light background on another device and it is a bit harder to see, still works ok though. What is different in case?

And we have to also keep it in sync with iOS, since on that it is the white background, so this has to change globally, not just locally.

And then we have different themes...

@davivel while we talk through this lets not hold up the release - we will slide this into the next release after we have agreement.

**Comment by jancborchardt** [comment]
@MTRichards the app tray on Android always has a light background by default. But regardless of the background or theme, the contrast of the cloud and the background is really low, which basically makes it look like a white square. It’s not iconic.

**Comment by stefan-niedermann** [comment]
I think this is fixed in https://github.com/owncloud/android/commit/2afd2d6fb6171e94fd40a712cab9ee036b4c8058, isn't it?

**Comment by AndyScherzinger** [comment]
It is still open in my opinion since the official/stable still uses the light/enterprise colored icon, the darkblue/white icon is only used for the beta app! There is no plan to move this to the stable release. We just did this so users can easily distinguish between both in case a user installed both in parallel.

But we can close it since no decision has been reached within way more than a year which basically mean the decision is stay with the enterprise one... (since no decision is also a decision!)

**Comment by AndyScherzinger** [comment]
To also provide my own opinion: We should move to the dark blue version matching the OSS Server side design (Also looks way better in the app switcher on Android since the bar then only shows the clouds :)

**Comment by Ricain** [comment]
I agree if it helps restarting debates :)

**Comment by jancborchardt** [comment]
Well, I still do agree. Did any further discussion happen here @MTRichards @karlitschek?

@MTRichards regarding the contrast: Even if the icon works well on a dark background, it still completely fails on a light one. And a proper icon should handle both with ease. Especially since the default app drawer background color is white since Android 5 already.

The contrast between white and light blue just is not very good and leads to our app not being very recognizable nor standing out.

**Comment by zhaofengli** [comment]
Hi, I've proposed a Material Design-compliant icon in #1594 which matches the OSS server-side design (dark blue).

**Comment by tripplex95** [comment]
i need help with importing owncloud using android studio 3.0 i am getting alot of error. i have downloaded the android project from here and i am getting alot of errors trying to build it. can anyone provide me with a working version or steps how to import it and build it in android studio.

**Comment by davigonz** [comment]
Hi @tripplex95 , thank you for reporting your problem but this issue is not related to it. Could you please open a new issue with further information? I will help you there.
